.gitignore

```diff
@@ -15,6 +15,9 @@ freqtrade/rpc/api_server/ui/*
 # Macos related
 .DS_Store
 
+# git files after merging
+*.orig
+
```

Comment:
Please fix your tooling to not keep backups.
That's a sign of a miss-configured tooling on your side (we recommend vscode or pycharm - neither of the two will write these files).

As such - there's no reason to exclude this.
Also make sure to remove all `.orig` files.

Reply:
https://stackoverflow.com/questions/12366150/how-to-delete-orig-files-after-merge-from-git-repository#comment62315290_12366207

Isn't?

Reply:
I don't know what tooling you use that it keeps .orig files (i've never seen that, and i've used quite some merge-tools over the years).

that thread seems to suggest using `git config --global mergetool.keepBackup false` (https://stackoverflow.com/a/34149790/1770304) - but i'm not sure if this fixes it as it's dependent on your setup.

Even if we were to change the gitignore (we're not) - this PR is clearly not the right place for such a change.